### DISCUSSION TITLE
Yeast model

### DISCUSSION THREAD

**Discussion by yeast1234** [Metaboverse/Metaboverse#156] [Q&A]
Hi,
I have a question regarding yeast as a model organism S.cerevisiae. There are many amino acid pathways missing in the Reactome db. Are you planning to include the KEGG pathway as well? Or is there any possibility to do so? That would really help me :) thanks in advance!

**Comment by j-berg** [comment]
I'm going to copy and paste my email response just in case it is useful info for anyone else:

Because KEGG does not have as permissive of a license as Reactome, the default is Reactome. However, if there are some KEGG database pathway files that you specifically want to use, we could work on making sure those are compatible with the custom network upload. Is there a URL to a KEGG pathway(s) you wanted to analyze your data against? I can take a look at it and see how much work it would require to format. This is something that I hoping to make more robust in the future anyways, so that people don't have to rely only on Reactome (it just makes it easier for commercial users to use the software for free).

FYI - I am tracking this general feature improvement with the following issues:
- https://github.com/Metaboverse/Metaboverse/issues/135
- https://github.com/Metaboverse/Metaboverse/issues/140
(as you can see, it will take me a bit to get to fully addressing these needs - I am the only one currently maintaining this right now and the priority when I do have some spare time is bug fixes)
